pallets/subtensor/src/lib.rs

```diff
@@ -1501,6 +1501,17 @@ pub mod pallet {
     pub type ImmuneOwnerUidsLimit<T> =
         StorageMap<_, Identity, NetUid, u16, ValueQuery, DefaultImmuneOwnerUidsLimit<T>>;
 
+    #[pallet::type_value]
+    /// Default validator cut 50%
+    pub fn DefaultValidatorCut<T: Config>() -> u16 {
+        u16::MAX / 2
```

Comment:
this should default to 100% else you're just forcing a massive change on all the subnet owners

Reply:
You’re right that makes sense, I misunderstood the default setting.